EditorPluginExample/Resources/EditorData/Example.plugin.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference path="../../typings/Atomic/AtomicWork.d.ts" />
+/// <reference path="../../typings/Atomic/EditorWork.d.ts" />
+
+const ExamplePluginUILabel = "Example Plugin";
+const ExamplePluginTBPath = "EditorData/Example.tb.txt";
+
+class ExamplePluginService implements Editor.HostExtensions.HostEditorService, Editor.HostExtensions.ProjectService {
+
+    name: string = "ExampleService";
+    description: string = "This service demonstrates plugin functionality functionality.";
+
+    private serviceLocator: Editor.HostExtensions.HostServiceLocator = null;
+    private extensionWindow: Editor.Modal.ExtensionWindow = null;
+
+    private helloLabel: Atomic.UITextField;
+    private nameField: Atomic.UIEditField;
+
+    initialize(serviceLoader: Editor.Extensions.ServiceLoader) {
```

Comment:
Even though the interface calls for ServiceLoader? Interesting thanks.

I've also noticed that despite us not explicitly implementing UIService in the class, it's accepted as input to the UIServiceRegistry.register. So interfaces definitely aren't as strict as I'm accustomed to in other languages